logstash/loki.conf

```diff
@@ -0,0 +1,28 @@
+input {
+#  generator { message => "Hello world!" count => 1 }
+  beats {
+    port => 5044
+  }
+}
+
+output {
+  loki {
+    url => "http://localhost:3100"
+
+    include_labels => ["@version"]
```

Comment:
maybe host and filename are better default ?

Reply:
I think prefer that we tried to support @beat.version instead of @beat_version. I'm wondering if there's code from logstash that we could use to extract values instead of building our own ? Could you investigate.

Finally I think we should have at least one default include_labels, may be @metadeta.beat ? something that tells the users it's coming from logstash. If nothing from field is interesting then may be we could add a extra label. client=logstash.

Reply:
I actually checked this before. Every logstash plugin has their own code to handle extracting data.  Filebeat is just one of the input plugin. There are other input plugins as well like `file input`, `kafka` etc.. I like the idea that we should include the metadeta fields as default labels. 

> I think prefer that we tried to support @beat.version instead of @beat_version
We can support `@beat.version`. However, I would still prefer to have underscore since that is the general convention in prometheus and loki labels.